### ISSUE TITLE
Use inline thread variable definition in all threads

### ISSUE THREAD

**Issue by jeromehue** [SpaceTeam/STS1_COBC_SW#24]
**Description:**
Define threads like this
```cpp
class EduHeartbeatThread : public RODOS::StaticThread<stackSize>
{
public:
 ...

private:
 ...
} eduHeartbeatThread;
```

Instead of this :
```cpp
class EduListenerThread : public StaticThread<>
{
public:
  ...
private:
  ...
};
auto const eduListenerThread = EduListenerThread();
```

**Files:**
EduListenerThread.cpp (only file as far as I can see)

**Comment by jeromehue** [comment]
Fixed by commit 021f33c for EduListenerThread.cpp

**Comment by PatrickKa** [comment]
Merged by #18

**Comment by PatrickKa** [comment]
We forgot about the threads in the tests like, e.g., `Spi.test.cpp` 🙈.
